src/ratingservice/main.py

```diff
@@ -77,6 +78,43 @@ def makeResult(data):
 #
 
 
+@app.route('/ratings', methods=['GET'])
+def getRatings():
+    '''Gets a list of all ratings.
+
+    Returns:
+        HTTP status 200 and Json payload { ratings: [{'id': (string), 'rating': (number)}] }
+        HTTP status 500 when there is an error querying DB or no data
+    '''
+
+    conn = getConnection()
+    if conn == None:
+        return makeError(500, 'failed to connect to DB')
+    try:
+        with conn.cursor() as cursor:
+            cursor.execute("SELECT eid, ROUND(rating,4) FROM ratings")
+            result = cursor.fetchall()
+        conn.commit()
+        print("1")
+        if result != None:
```

Comment:
nit: `if result is not None` [is preferred](https://stackoverflow.com/questions/3257919/what-is-the-difference-between-is-none-and-none)

Or `if not ratings` if you want empty lists to also skip this block